playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/containerized_upgrade.yml

```diff
@@ -0,0 +1,6 @@
+- name: Update system_units
+  script: ../files/ensure_system_units_have_version.sh {{ openshift.common.service_type }} {{ openshift.common.deployment_type }} {{ g_new_version }}
+
+- name: Ensure python-yaml present for config upgrade
+  action: "{{ ansible_pkg_mgr }} name=PyYAML state=present"
+  when: not openshift.common.is_atomic | bool
```

Comment:
This shouldn't be needed here... PyYAML would only be needed for running the config update module.